tests/GraphQl/Resolver/Factory/CollectionResolverFactoryTest.php

```diff
@@ -165,7 +165,7 @@ public function testCreatePaginatorCollectionResolver()
         $resolveInfo = new ResolveInfo('relatedDummies', [], null, new ObjectType(['name' => '']), '', new Schema([]), null, null, null, null);
 
         $this->assertEquals(
-            ['edges' => [['node' => 'normalizedObject1', 'cursor' => 'Mg==']], 'pageInfo' => ['endCursor' => 'MTY=', 'hasNextPage' => true], 'totalCount' => 17],
+            ['edges' => [['node' => 'normalizedObject1', 'cursor' => 'Mg==']], 'pageInfo' => ['endCursor' => 'OQ==', 'hasNextPage' => true], 'totalCount' => 17.],
```

Comment:
```suggestion
            ['edges' => [['node' => 'normalizedObject1', 'cursor' => 'Mg==']], 'pageInfo' => ['endCursor' => 'OQ==', 'hasNextPage' => true], 'totalCount' => 17],
```

Any reason for the `.` at the end?

Reply:
To avoid this kind of message from PHPUnit:
```diff
--- Expected
+++ Actual
@@ @@
 Array (
     'edges' => Array (...)
     'pageInfo' => Array (
-        'endCursor' => 'MTY='
+        'endCursor' => 'OQ=='
         'hasNextPage' => true
     )
-    'totalCount' => 17
+    'totalCount' => 17.0
 )
```

Reply:
Makes sense, and I expected something like that. Just wanted to make sure, since that change was not in the original PR and it seemed odd at first 🙂